src/comitNode.ts

```diff
@@ -82,6 +87,27 @@ export class ComitNode {
     return request(options).then(response => response.entities);
   }
 
+  public getMetadata(): Promise<ComitMetadata> {
+    const options = {
+      method: "GET",
+      url: this.config.prependUrlIfNeeded("/").toString(),
+      json: true
+    };
+
+    return request(options).then(response => {
+      // TODO: cnd should return its public address. This is a workaround it's currently not returned. Related issue is: #79
```

Comment:
Huh?
I thought you were saying the opposite in the chat?
Is this an artifact from before our discussion?

Why sneak in the value from the config here instead of just ignoring the whole array from the request and grab the value from the config file where you need it?
I think this method should return whatever it gets from `cnd` an not trying to be clever in what other parts of the code need.